src/compiler/scala/tools/nsc/settings/MutableSettings.scala

```diff
@@ -455,6 +455,7 @@ class MutableSettings(val errorFn: String => Unit)
     type T = Boolean
     protected var v: Boolean = false
     override def value: Boolean = v
+    def boolValue: Boolean = v
```

Comment:
OK, boxing, thanks. It would take years of research to determine a good name. Maybe `asBoolean`.